guides/concepts/exchange.md

```diff
@@ -17,9 +17,34 @@ If the offer doesn't cross an existing offer, the offer is saved in the orderboo
 
 Starting in protocol version 10, it is no longer possible for an offer to be invalidated because the account owning the offer no longer has the asset for sale. Each offer contributes selling liabilities for the selling asset and buying liabilities for the buying asset, which are aggregated in the account (for lumens) or trustline (for other assets) owned by the account creating the offer. Any operation that would cause an account to be unable to satisfy its liabilities, such as sending away too much balance, will fail. This guarantees that any offer in the orderbook can be executed entirely.
 
-Offers in Stellar behave like limit orders in traditional markets. 
+Offers in Stellar behave like limit orders in traditional markets.
 
-For offers placed at the same price, the older offer is filled before the newer one.  
+For offers placed at the same price, the older offer is filled before the newer one.
+
+### Price
+
+Each offer in Stellar has an associated price with it, which is represented as a ratio of the
+two assets in the offer. This is to ensure there is no loss of precision when representing the
+price of the offer (as opposed to storing the fraction as a floating-point number).
+
+Prices are specified as a `{numerator, denominator}` pair, with both components of the fraction
+represented as 32 bit signed integers. When expressing a price of "Asset A in terms of Asset B",
+the amount of B is always the numerator, and A is always the denominator.
+
+When creating a "buy"/"bid" offer in Stellar via the [Manage Buy
+Offer](./list-of-operations.md#manage-buy-offer) operation, the price is specified as 1 unit of
+what you're _buying_ in terms of what you're _selling_. For example, if you were _buying_ 20 USD in
+exchange for 100 XLM, you would specify the price as `{100, 20}`, which would be the equivalent of
+1 USD @ 5 XLM.
+
+When creating a "sell"/"offer"/"ask" offer in Stellar via the [Manage Sell
+Offer](./list-of-operations.md#manage-sell-offer) operation, the price is specified as 1
+unit of what you're _selling_ in terms of what you're _buying_. For example, if you were _selling_
+10 USD in exchange for 100 XLM, you would specify the price as `{100, 10}`, which would be the
+equivalent of 1 USD @ 10 XLM (_nice profit_).
+
+It's important to note that the price is set _after fees_, so be sure to factor that into how you
```

Comment:
Is that true? I thought the price was before fees. That is, there's a transaction fee to place the offer, which is added to the price you pay for the trade. The some of those two is your total price (although often you can't sum them because they're different units). I'd also be more specific about the fees, and point out that it's just the standard transaction fee. That's just my current understanding, I may have totally misunderstood.

Reply:
I can definitely make it more specific, but I'm also confused. I got that information from the XDR: https://github.com/stellar/stellar-core/blob/aecb754bf4e17eb58545d85730ea1e5464104eda/src/xdr/Stellar-ledger-entries.x#L227

@jonjove can you verify the actual behavior?

Reply:
Hrm I'm now confused too. I think it's a bit of a language issue. It'd be more clear to me to say the price includes fees or excludes fees rather than before or after. We'll see which is right from Jon!

Reply:
I agree with @tomquisel that this is a language issue. I think we should update the language to emphasize that the fee you pay and the price you pay are completely independent concepts, rather than trying to capture this notion with terms like before/after/includes/excludes. It might also help to include some explanations for why it _is reasonable_ for it to work this way: that the fee relates to the transaction not any specific operation, and that the fee and the price might involve different assets.

An example or two would probably help to clarify the precise meaning. For example: buying 10 X for 5 native (price 1/2) with a fee of 1000 native stroops leads to a total gain of 10 X and total loss of 5 native + 1000 native stroops. This result is reached because you pay a fee of 1000 native stroops to execute your transaction, and then you receive 2 X for every 1 native you send when your offer executes.